shell/platform/darwin/ios/framework/Source/FlutterView.mm

```diff
@@ -30,4 +36,87 @@ - (BOOL)enableInputClicksWhenVisible {
   return YES;
 }
 
+void SnapshotRasterizer(ftl::WeakPtr<shell::Rasterizer> rasterizer,
+                        CGContextRef context,
+                        bool is_opaque) {
+  if (!rasterizer) {
```

Comment:
:D From the style guide "curly braces are not required for single-line statements, but they are allowed if you like them;" Doesn't say what to do when you no feelings about it whatsoever. I think i'll just leave it as is.